tutorials/CMakeLists.txt

```diff
@@ -83,6 +83,10 @@ if(NOT ROOT_xml_FOUND)
                 unfold/*.C)        # unfold requires xml
 endif()
 
+if(NOT XROOTD_FOUND)
+  set(dataframe_veto  dataframe/df102_NanoAODDimuonAnalysis.C)
```

Comment:
Shouldn't this be `xrootd_veto`? Isn't there something we can append to instead of creating a new veto list variable?

Reply:
I haven't found something which captures the case of a non-existent xrootd. However, the naming is a valid point!

Reply:
I've changed it!